src/h5web/providers/mock/values.ts

```diff
@@ -65,42 +65,42 @@ export const mockValues = {
   threeD_cplx: [
     [
       [
-        [0, 0],
-        [5, 0],
-        [10, 5],
-        [0.1, 0.5],
+        [2, 0],
+        [1.41, 1.41],
+        [1, 1.73],
+        [0, 2],
       ],
       [
-        [-1, -5],
-        [-0.1, -0.8],
-        [2, 7],
-        [0.9, -5],
+        [4, 0],
+        [2.82, 2.82],
+        [2, 3.46],
+        [0, 4],
       ],
       [
-        [0, 0],
-        [5, 2],
-        [0, -2],
-        [-0.5, 2],
+        [1, 0],
+        [0.71, 0.71],
+        [0.5, 0.87],
+        [0, 1],
       ],
     ],
     [
       [
-        [0.0001, 0],
-        [0, 0.0001],
-        [2, 8],
-        [0.7, 0.7],
+        [-1, 0],
+        [0.87, -0.5],
+        [0.92, -0.38],
+        [1, 0],
       ],
       [
-        [0.5, -0.5],
-        [4, 2],
-        [0.4, 4.4],
-        [2.3, 3.2],
+        [-4, 0],
+        [3.46, -2],
+        [3.7, -1.53],
+        [4, 0],
       ],
       [
-        [9.5, 1.4],
-        [5, 0],
-        [0, -2],
-        [8, 2.1],
+        [-8, 0],
+        [6.93, -4],
+        [7.39, -3.06],
+        [8, 0],
```

Comment:
I changed the data so that I can have slices of same phases but different amplitudes